pkg/services/object/server.go

```diff
@@ -2071,7 +2070,7 @@ func (s *server) processSearchRequest(ctx context.Context, req *protoobject.Sear
 			return nil, fmt.Errorf("merge results from container nodes: %w", err)
 		}
 		if more {
-			if newCursor, err = meta.CalculateCursor(fs, res[len(res)-1]); err != nil {
+			if newCursor, err = objectcore.CalculateCursor(fs, res[len(res)-1]); err != nil {
```

Comment:
shouldnt be triggered by the commit changing tests

Reply:
why? commit message has some words about it. without moving it, we have cyclic deps, it must be moved. all I can suggest is moving in a separate commit (why? commit is for sharing only, it changes nothing, and `CalculateCursor` is shared in it)

Reply:
ok lets have it. It confuses very much when code is changed for test purposes in most of the cases